Assets/Scripts/Character/Abilities/PsychicScream.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PsychicScream : ISkill
+{
+    private SkillSettings settings;
+    private MonoBehaviour mono;
+
+    private float cooldownTimestamp;
+
+    public PsychicScream(SkillSettings settings, MonoBehaviour mono)
+    {
+        this.settings = settings;
+        this.mono = mono;
+    }
+
+    public void Execute(Animator animator)
+    {
+        if (!IsOnCooldown())
+        {
+            cooldownTimestamp = Time.time + settings.cooldown;
+            float maxDistance = 10;
+            List<GameObject> enemiesInRange = new List<GameObject>();
+            foreach (GameObject enemy in GameObject.FindGameObjectsWithTag(Constants.ENEMY_TAG))
```

Comment:
Can you move this to start? The amount of in game enemies is not going to change so... so might be performance cheaper to do this only once - also don't need to call new. just enemiesInRange = gameObject.FindGameObjectsWithTag ... etc

Reply:
I think since this is no MonoBehaviour there is no start method to put this code in. but i will change it so that its only executed once.